nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/NotificationService.java

```diff
@@ -190,7 +192,9 @@ public void shutdownNotificationService() {
         if (execution != null) {
             execution.cancel(true);
         }
-        notifiers.forEach(NotifierHandler::destroy);
+        if (enabled) {
```

Comment:
Not so sure about this guard. If `notifiers` isn't empty shouldn't the `destroy` run even when not disabled? What if I do the bootstrap, disable the service and then get here, shouldn't we still `destroy`?